.github/workflows/report-merged-pr.yml

```diff
@@ -0,0 +1,33 @@
+# Report PR merged event to Datadog
+
+name: Report Merged PR
+
+on:
+  pull_request:
+    types: closed
+
+permissions: {}
+
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+        with:
+          fetch-depth: 0
```

Comment:
do you really need to clone with full depth?

Reply:
Don't know actually! I copy-pasted from elsewhere. Cloning to have the invoke task, checking now what fetch depth means. Do you have suggestions?